docs/source/guide/configuration.rst

```diff
@@ -61,64 +61,57 @@ Boto3 supports using proxies as intermediaries between your code and AWS. Proxie
 Specifying proxy servers
 ''''''''''''''''''''''''
 
-You can specify proxy servers to be used when communicating between specific endpoints, or when
-using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary that maps
-the protocol name or endpoint URL to the address of a proxy server.
+You can specify proxy servers to be used for connections when using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary that maps the protocol name to the address and port number of a proxy server.
 
 .. code-block:: python
 
     import boto3
     from botocore.config import Config
 
-    proxy_list = {
-        'http': 'proxy.example.com:6502',
-        'http://mysite.org': 'proxy.example.org:2010'
+    proxy_definitions = {
+        'http': 'https://proxy.amazon.com:6502',
```

Comment:
This would need to be an http scheme address since this connection can't negotiate TLS.